revolution/motor.py

```diff
@@ -6,6 +6,8 @@
 from revolution.environment import Endpoint
 from revolution.worker import Worker
 
+import numpy as np
```

Comment:
The ordering of imports are... ``stdlib``, empty line, ``third party``, empty line, ``local code``. In other words, ``import numpy as np`` should go between ``stdlib`` and ``from revolution...`` block.